apps/nextjs-website/react-components/components/Footer/components/FooterColumn.tsx

```diff
@@ -0,0 +1,73 @@
+import { Link, Stack, Typography } from '@mui/material';
+import { hrefNoOp } from '../../../types/common/Common.types';
+import { LogoPagoPACompany } from '../assets/LogoPagoPACompany';
+import { EIcon } from '../../../components/common/EIcon';
+import { FooterColumnProps } from '../../..//types/Footer/Footer.types';
+import { Button } from '@mui/material';
+
+export const FooterColumn = ({
+  data,
+  companyLink,
+  icons,
+}: FooterColumnProps) => (
+  <Stack spacing={2} display='flex' alignItems={{ xs: 'center', sm: 'start' }}>
+    {data?.title && <Typography variant='overline'>{data.title}</Typography>}
```

Comment:
```suggestion
    {data.title && <Typography variant='overline'>{data.title}</Typography>}
```
`data` isn't optional